### DISCUSSION TITLE
AttributeError: 'NoneType' object has no attribute 'id'

### DISCUSSION THREAD

**Discussion by Monim100** [apache/airflow#30906] [General]
### Apache Airflow version

Other Airflow 2 version (please specify below)

### What happened

Currently upgraded to airflow 2.0.2 from 1.10. When trying to login to the server I get the following error:
![image](https://user-images.githubusercontent.com/127773874/234849902-32d1a140-7da9-4527-b912-cf221c608469.png)
Searched online and haven't found a fix or solution to it. 

### What you think should happen instead

I would like to be able to login. 

### How to reproduce

using airflow 2.0.2 and running it locally in a docker container.

### Operating System

Ubuntu WSL

### Deployment

Official Apache Airflow Helm Chart

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template! If you are willing to raise PR to address this issue please do so, no need to wait for approval.

**Comment by potiuk** [comment]
Oh yes. Similar issues were fixed (likely big number of them) since we released 2.0.2 about 2.5 years ago. Since then there were plenty of bugfix and feature relases - likely 20 or so (and you can take a look at the release notes).

Did you miss the releases were done maybe?

**Comment by potiuk** [comment]
Just in case - here are the release notes you can browse:

https://airflow.apache.org/docs/apache-airflow/stable/release_notes.html

Also you can search for similar issues in our Github Issues, Discussons and PRs if you are curious.

Good luck with upgrading!
